Assets/Scripts/CrumblingFloorEvent.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CrumblingFloorEvent : MonoBehaviour
+{
+    public float crumbleDelay;
+
+    public AudioClip crumbling;
+
+    public AudioClip crumblingWarning;
+
+    private float timerCountDown = 1;
+
+    bool isColliding = false;
+
+    bool hasCrumblingAudioPlayed = false;
+
+
+    private void Update()
+    {
+        if (isColliding == true)
+        {
+            timerCountDown -= Time.deltaTime;
+            if (timerCountDown < 0)
+            {
+                timerCountDown = 0;
+            }
+        }
+    }
+
+    private void OnTriggerEnter2D(Collider2D collision)
+    {
+        if (collision.gameObject.tag == "Player")
+        {
+            Debug.Log("player has come in contact with the platform");
```

Comment:
id appreciate it if you comment out the print statements before you do a pull request so it doesn't clutter up the debug log for others.